js/Create_Phone_Number.js

```diff
@@ -1,11 +1,13 @@
 // https://www.codewars.com/kata/create-phone-number/train/javascript
 
 function createPhoneNumber(numbers){
-    var numbersArray = numbers;
-    console.log('start');
-    if (numbersArray.length === 10){
-      return ('('+numbersArray[0]+numbersArray[1]+numbersArray[2]+')'+' '+numbersArray[3]+numbersArray[4]+numbersArray[5]+'-'+numbersArray[6]+numbersArray[7]+numbersArray[8]+numbersArray[9]);
-      
-    }
+  if (numbers.length === 10){
+    const a = numbers.slice(0,3).join('');
+    const b = numbers.slice(3,6).join('');
+    const c = numbers.slice(6,10).join('');
+    return (`(${a})${b}-${c}`);
+    
+  } else {
+    return console.log('Array length is not equal 10')
```

Comment:
What is the return value of `createPhoneNumber` when an incorrect input was passed in? How will the caller of the function know that it was incorrect?

Reply:
What should i do ? Show pop-up or return string with info ?

Reply:
There is a language mechanism for returning errors when something goes wrong in a function :smile: 

There are other ways :smile: As a consumer of such a function, how would you like to know that something went wrong?

Reply:
I will return error message :+1:  I will change that